### ISSUE TITLE
Typescript project runs with a type error in serve mode (lack of shallow funciton)

### ISSUE THREAD

**Issue by anton-tarasov-qs** [vuejs/vue-cli#1309]
### Version
3.0.0-beta.10

### Reproduction link
[https://www.dropbox.com/s/t15zyqygx17aett/chrome_2018-05-16_20-47-21.png](https://www.dropbox.com/s/t15zyqygx17aett/chrome_2018-05-16_20-47-21.png)

### Steps to reproduce
1. install vue-cli 3.0.0-beta.10
2. create a Typescript project with Jest unit-tests
3. npm run serve

### What is expected?
Development server runs without errors

### What is actually happening?
It shows an error:

```
ERROR in .../tests/unit/HelloWorld.spec.ts
1:10 Module '".../node_modules/@vue/test-utils/types/index"' has no exported member 'shallow'.
import { shallow } from '@vue/test-utils';
```

---
`shallow` function was renamed to `shallowMount` in `@vue/test-utils` types file

**Comment by amritk** [comment]
Looks like it was fixed in this commit, https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-plugin-unit-jest/generator/template/tests/unit/HelloWorld.spec.js
but the issue still exists.

**Comment by JamesCrann** [comment]
Even if you change it to shallowMount the test still wont pass.

![image](https://user-images.githubusercontent.com/2524952/40178400-bd5b736a-5a24-11e8-9974-0c50e87f097b.png)

**Comment by Tmr** [comment]
@JamesCrann Please note the line 
```
import "core-js/modules/es6.promise";
^^^^^^
```
in your output. The source of failure in HelloWorld.vue on line 2.

**Comment by yyx990803** [comment]
As said this is fixed in 5c54df7 and will be released soon. @JamesCrann that seems to be a different issue, please open with a separate repro.
